jaggeryapps/admin/templates/header.jag

```diff
@@ -299,7 +300,36 @@ if (user == null) {
     <%}%>
     });
 </script>
-
+<%
+    if(googleAnalytics.enabled) {
+%>
+    <script>
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function()
+        { (i[r].q=i[r].q||[]).push(arguments)}
+
+        ,i[r].l=1*new Date();a=s.createElement(o),
+        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+        ga('create', '<%=googleAnalytics.trackerId%>', 'auto');
+        ga('send', 'pageview');
+
+        </script>
+
+        <!-- Google Tag Manager -->
```

Comment:
We might not need "Tag manager" Need to check this and remove if it is the case.